File: java/BurbleSort.java
public class BurbleSort {
  public static void main(String[] args) {
    int[] arr = {2,42,63,36,73,15,37,58,14,63,61,27,82,35,31};
    for(int i = 0; i < arr.length; i++){
      for (int j = 0; j < (arr.length - i - 1); j++){
        if (arr[j+1] < arr[j]) {
          int temp = arr[j];
          arr[j] = arr[j+1];
          arr[j+1] = temp;
        }
      }
      System.out.print("[");
      for(int e:arr) {
      System.out.print(e + ",");
      }
      System.out.print("]\n");
    }
  }
}

File: js/burble-sort.js
let arr = [24,24,36,2,2,64,47,6,2,12,24,5,69,52,4,43,5,1,24,8,96,81,]

for (let i = 0; i < arr.length; i++) {
  for (let j = 0; j < (arr.length - i - 1); j++ ) {
    if (arr[j+1] < arr[j]) {
      let temp = arr[j];
      arr[j] = arr[j+1];
      arr[j+1] = temp;
    }
  }
  console.log(arr);
}
